README.md

```diff
@@ -42,24 +42,22 @@ The application will be available at: http://127.0.0.1:5003/ or http://localhost
 Navigate to Your Local Repository:
 
 ```
-cd path/to/your/local/repo
+cd path\to\csub_fall2024_cse6550_team3
```

Comment:
I'm unsure about this

Reply:
The path is different for every one i think, so i have given sample command.